canvas_modules/harness/src/client/constants/json/functionlist.json

```diff
@@ -6338,6 +6338,8 @@
 			{
 				"id": "globals",
 				"field_columns": {
+					"add_column_info": {
```

Comment:
Why is this needed? We shouldn't need to change the schema for this extra column.